src/api/entitlements-test.js

```diff
@@ -84,4 +84,15 @@ describes.realWin('Entitlements', {}, () => {
       );
     });
   });
+
+  describe('consume', () => {
+    it('executes callback', () => {
+      const consumeSpy = sandbox.spy();
```

Comment:
nit: do you want to rename "consumeSpy" to "callbackSpy"?

currently, it was confusing to me at an initial glance since the method under test is also called "consume".

Reply:
Oh yeah, that was confusing :joy:  - I updated the naming to "consume handler" / "consumeHandlerSpy"